third_party/move/move-vm/runtime/src/loader/mod.rs

```diff
@@ -2504,3 +2485,19 @@ fn match_return_type<'a>(
         | (Type::Reference(_), _) => false,
     }
 }
+
+pub(crate) fn check_natives(module: &CompiledModule) -> VMResult<()> {
+    // TODO: fix check and error code if we leave something around for native structs.
```

Comment:
is this an old todo? do you follow what it wants?

Reply:
Old todo. I think there were some native struct types (like native functions) but they got removed, though still stayed in file format. In theory, deserializer should ensure that it fails when such a native struct is constructed, but I think it is better be done in some other PR on main.

Reply:
sure, just wasn't sure and didn't follow. makes sense.